### PR TITLE
add foter link

### PR DESCRIPTION
**Pull request by SergiusHp**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -95,13 +95,13 @@ <h2>$49</h2>/month
                 <div class="foter_block_end_1">
                     <div class="foter_block_end_img"></div>
                     <div class="foter_block_end_social-icon">
-                        <div><i class="fab fa-facebook color-icon"></i></div>
-                        <div><i class="fab fa-twitter color-icon"></i></div>
-                        <div><i class="fab fa-google color-icon"></i></div>
+                        <div ><a href="/jpg/mini_2.png"><i class="fab fa-facebook color-icon"></i></a></div>
```
Comment (lapc002) [inline index.html]: please change to correct url . for example for facebook => https://facebook.com etc.

File: style.css [hunk 2]
```diff
@@ -193,22 +193,46 @@
 }
 
 .foter_block_item-text {
-    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
```
Comment (lapc002) [inline style.css]: font-family should be on the top of document

File: index.html (hunk 1 above)
Comment (SergiusHp) [reply index.html]: done

File: style.css (hunk 2 above)
Comment (SergiusHp) [reply style.css]: done

**Review by lapc002 — APPROVED** [review]

**Pull request merged**